app/site.rb

```diff
@@ -217,21 +223,31 @@ def makefile?
 	protected
 
 		def maybe_commit(git, message, user)
-			git.status.each do |f|
-				if f.type
-					if user && user.email
-						git.commit("#{message} via tint", author: "#{user.fn} <#{user.email}>")
-					else
-						git.commit("#{message} via tint")
-					end
+			begin
+				status = git.status
+			rescue Git::GitExecuteError
```

Comment:
So status fails if there are no commits, then we rescue and just commit since there is nothing to loop over? Am I understanding that correctly?